test/e2e/snaps/test-snap-ethprovider.spec.js

```diff
@@ -47,6 +47,9 @@ describe('Test Snap ethereum_provider', function () {
           tag: 'button',
         });
 
+        // wait and scroll if necessary
+        await driver.clickElementSafe('[data-testid="snap-install-scroll"]');
+
```

Comment:
The test is failing because the increased font size shifts the content downward, triggering the scrollbar and button visibility, which disables the Confirm button. Unlike other snap e2e tests, this one doesn't check for the scrollbar. Adding that check here resolves the issue.

![Screenshot 2025-04-02 at 10 29 29 AM](https://github.com/user-attachments/assets/72746a70-755f-41b4-8f7a-a48752cc9fc7)